src/main/resources/application.properties

```diff
@@ -12,17 +12,15 @@ server.servlet.contextPath=/Geoweaver
 logging.level.root=FATAL
 logging.level.org.springframework=FATAL
 logging.level.com.gw=FATAL
-logging.level.org.hibernate=FATAL
+logging.level.org.hibernate=DEBUG
```

Comment:
We really don't care about hibernate log in production. Can you turn this back to FATAL?